varya/default-style/variables-editor.css

```diff
@@ -37,37 +37,37 @@
 
 	/* Branding */
 	--branding--title--font-size: var(--heading--font-size-h1);
+	--branding--title--font-weight: normal;
 
 	/* Navigation */
 	--primary-nav--font-family: var(--global--font-secondary);
 	--primary-nav--font-size: 16px;
 	--primary-nav--font-weight: normal;
 
 	/* Blocks */
-	--button--border-radius: 4px;
```

Comment:
The CSS-variables for the editor are the exact same one used in the frontend. When resetting variables, we have to manually keep them in sync but there’s probably a way to have this done automatically with Sass.